usmqe_tests/api/grafana/test_volume_dashboard.py

```diff
@@ -0,0 +1,107 @@
+"""
+REST API test suite - Grafana dashboard volume-dashboard
+"""
+
+import pytest
+from usmqe.api.grafanaapi import grafanaapi
+
+
+LOGGER = pytest.get_logger('volume_dashboard', module=True)
+"""@pylatest default
+Setup
+=====
+
+Prepare USM cluster accordingly to documentation.
+``GRAFANA`` for this file stands for Grafana API url used by tested Tendrl
+server.
+``GRAPHITE`` for this file stands for Graphite API url used by tested Tendrl
+server.
+
+"""
+
+"""@pylatest default
+Teardown
+========
+"""
+
+
+def test_layout():
+    """@pylatest grafana/layout
+    API-grafana: layout
+    *******************
+
+    .. test_metadata:: author ebondare@redhat.com
+
+    Description
+    ===========
+
+    Check that layout of dashboard is according to specification:
+    ``https://github.com/Tendrl/specifications/issues/224``
+    """
+    api = grafanaapi.GrafanaApi()
+    """@pylatest grafana/layout
+    .. test_step:: 1
+
+        Send **GET** request to:
+        ``GRAFANA/dashboards/db/volume-dashboard``.
+
+    .. test_result:: 1
+
+        JSON structure containing data related to layout is returned.
+    """
+    layout = api.get_dashboard("volume-dashboard")
+    pytest.check(
+        len(layout) > 0,
+        "volume-dashboard layout should not be empty")
+
+    """@pylatest grafana/layout
+    .. test_step:: 2
+
+        Compare structure of panels and rows as defined in specification:
+        ``https://github.com/Tendrl/specifications/issues/224``
+
+    .. test_result:: 2
+
+        Defined structure and structure from Grafana API are equivalent.
+    """
+    structure_defined = {
+        'At-a-Glance': [
+            'Health',
+            'Bricks',
+            'Brick Status',
+            'Subvolume',
+            'Geo-Replication Sessions',
+            'Rebalance',
+            'Rebalance Status',
+            'Snapshots'],
+        'Capacity': [
+            'Capacity Utilization',
+            'Capacity Available',
+            'Weekly Growth Rate',
+            'Weeks Remaining',
+            'Capacity Utilization'],
+        'Performance': [
+            'IOPS',
+            'LVM Thin Pool Metadata %',
+            'LVM Thin Pool Data Usage %'],
+        'Profiling Information': [
+            'Top File Operations',
+            'File Operations For Locks',
+            'File Operations for Read/Write',
+            'File Operations for Inode Operations',
+            'File Operations for Entry Operations']}
+    structure = {}
+    for row in layout["dashboard"]["rows"]:
+        structure[row["title"]] = []
+        for panel in row["panels"]:
+            if panel["title"]:
+                structure[row["title"]].append(panel["title"])
+            elif "displayName" in panel.keys() and panel["displayName"]:
+                structure[row["title"]].append(panel["displayName"])
+    print(structure)
```

Comment:
If you think this is useful to print out, use `LOGGER.debug()` instead of `print()`.